mbf_costmap_nav/src/mbf_costmap_nav/costmap_navigation_server.cpp

```diff
@@ -180,6 +180,9 @@ CostmapNavigationServer::CostmapNavigationServer(const TFPtr& tf_listener_ptr)
   find_valid_pose_srv_ =
       private_nh_.advertiseService("find_valid_pose", &CostmapNavigationServer::callServiceFindValidPose, this);
 
+  force_update_costmaps_srv_ =
```

Comment:
S: move clear_costmaps after this one, as a more logic grouping

Reply:
Done